challenge_stickler_for_rules/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
Filter per challenge
Dont include others

Reply:
Fixing it is a pain in the *ss. I don't know who fucked up the git history